src/topology/algebra/uniform_ring.lean

```diff
@@ -142,6 +145,33 @@ instance top_ring_compl : topological_ring (completion α) :=
 def map_ring_hom (hf : continuous f) : completion α →+* completion β :=
 extension_hom (coe_ring_hom.comp f) (continuous_coe_ring_hom.comp  hf)
 
+section algebra
+variables (R : Type*) [ring R] [uniform_space R] [uniform_add_group R] [topological_ring R]
+  (S : Type*) [comm_semiring S] [algebra S R] [has_uniform_continuous_const_smul S R]
```

Comment:
Can you rename the variables in this section from `S R` to `R A`, since that matches what we normally do for `algebra`, and also matches what you wrote in the PR description?